### DISCUSSION TITLE
LOST FUND ON DEGEN BRIDGE

### DISCUSSION THREAD

**Discussion by dgl16** [blockscout/blockscout#9764] [Support] [answered]
I bridged some ETH from base network to degen but the funds doesnt apepar on my degen wallet, i would be grateful to receive some help. tx hash: 0xd8bcf814681789658b4cbb8a35602aa25ce9e105e262ce3bd7ca3be3993eb071

**Comment by natancejasgarcia** [comment]
hi, do u have your eth now?, i have the same problem

**Reply by dgl16** [reply, marked as answer]
Hi, looks like the bridge is saturated but funds are safe, they’re giving answers on the tg chat https://t.me/degentokenbase

**Reply by natancejasgarcia** [reply]
thx men!
